modules/integration/src/test/scala/scala/cli/integration/BloopTests.scala

```diff
@@ -98,4 +95,23 @@ class BloopTests extends munit.FunSuite {
     }
   }
 
+  test("bloop exit works") {
+    def assertBloopNotRunning(): Unit = {
+      val javaProcesses = os.proc("jps", "-l").call().out.text()
+      expect(!javaProcesses.contains("bloop.Bloop"))
+    }
+
+    val inputs = TestInputs(Seq.empty)
+    inputs.fromRoot { _ =>
+      BloopUtil.killBloop()
+      TestUtil.retry()(assertBloopNotRunning())
+
+      val res = runScalaCli("bloop", "start").call()
+      assert(res.exitCode == 0, clues(res.out.text()))
```

Comment:
No need to check `res.exitCode`, os-lib checks it by default (unless we pass `check = false` to `call()`).